packages/SwingSet/tools/bundleTool.js

```diff
@@ -30,7 +31,7 @@ export const makeFileWriter = (fileName, { fs, path }) => {
   });
 };
 
-export const makeBundleCache = (wr, cwd, readPowers, opts) => {
+export const makeBundleCache = (wr, options, cwd, readPowers, opts) => {
```

Comment:
`options` _and_ `opts`? Any particular reason not to combine them?

Reply:
they're different bags. the first one is for bundleSource. but the latter isn't ever used so I'll remove it.